src/SprykerEco/Zed/Unzer/Persistence/Propel/Schema/spy_unzer.schema.xml

```diff
@@ -140,4 +140,15 @@
         </unique>
     </table>
 
+    <table name="spy_payment_unzer_shipment_charge" idMethod="native" allowPkInsert="true" isCrossRef="true">
```

Comment:
Just for the records, it was approved by Denis Turkov that we can release this as 1.0.1 because its basically a hotfix and because of the 1.0.0 release is so new, we are allowed to make an exception here.